DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -957,6 +957,30 @@ function ConEventAddGoal GetGoalConEvent(name goalName, name convname, optional
     return GetGoalConEventStatic(goalName, GetConversation(convname), which);
 }
 
+function AddNoteFromNoteEvent(ConEventAddNote event)
+{
+	if (!event.bNoteAdded) {
+		player().AddNote(event.noteText, False, True);
+		event.bNoteAdded = True;
+	}
+}
+
+function ConEventAddNote GetNoteEventFromConv(Conversation conv, optional string substring)
```

Comment:
we already have

`function DeusExNote AddNoteFromConv(#var(PlayerPawn) player, bool bEmptyNotes, name convname, optional int which)`

in `DXRModules\DeusEx\Classes\DXRStartMap.uc`

Reply:
What kind of nerd made an over-designed function like that?